routingv8/routes.go

```diff
@@ -0,0 +1,41 @@
+package routingv8
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Routes returns all possible routes between origin and destionation.
+// See https://developer.here.com/documentation/routing-api/dev_guide/topics/send-request.html#send-a-request
+// for details about other parameters.
+func (s *RoutingService) Routes(
+	ctx context.Context,
+	req *RoutesRequest,
+) (_ *RoutesResponse, err error) {
+	if req.TransportMode.String() == invalid {
+		return nil, fmt.Errorf("invalid transportmode")
+	}
+
+	u, err := s.URL.Parse("routes")
+	if err != nil {
+		return nil, err
+	}
+
+	values := make(url.Values)
+	values.Add("return", "summary")
+	values.Add("transportMode", req.TransportMode.String())
```

Comment:
I think we also need to check for `unspecified` TransportMode:
```suggestion
	tm := req.TransportMode.String()
	if tm == invalid || tm == unspecified {
		return nil, fmt.Errorf("invalid TransportMode")
	}
	u, err := s.URL.Parse("routes")
	if err != nil {
		return nil, err
	}

	values := make(url.Values)
	values.Add("return", "summary")
	values.Add("transportMode", tm)
```